pkg/apis/core/v1alpha1/types_overridepolicy.go

```diff
@@ -84,16 +84,70 @@ type TargetClusters struct {
 // Overriders contains a list of override patches.
 // The order in which the override patches take effect is:
 // - Image
+// - Command
+// - Args
+// - Annotations
+// - Labels
 // - JsonPatch
 type Overriders struct {
 	// Image specifies the overriders that applies to the image.
 	// +optional
 	Image []ImageOverrider `json:"image,omitempty"`
+
+	// Command specifies overriders that applies to the container commands.
```

Comment:
```suggestion
	// Command specifies overriders that apply to the container commands.
```
Pls fix it for all overrides

Reply:
The code has been updated.